src/dipdup/config.py

```diff
@@ -1155,7 +1154,9 @@ class TokenTransferHandlerConfig(HandlerConfig, kind='handler'):
 
     contract: ContractConfig | None = None
     token_id: int | None = None
-    from_: ContractConfig | None = Field(default=None, alias='from')
+    # FIXME: Can't use `from_` field alias in dataclass; fixed in dipdup.yaml instead
+    # FIXME: See https://github.com/pydantic/pydantic/issues/4286
+    from_: ContractConfig | None = None
```

Comment:
If pydantic.Field only purpose is to make an alias, you can use __getattr__, __setattr__ methods or use descriptor protocol class with __get__ __set__ methods, instead of cast function